<launch>
    <node name="road_segmentation" pkg="road_segmentation" type="road_segmentation_node" output="screen" args="_image_transport:=compressed" cwd="node">
        <param name="sub_camera_topic" value="/camera/rgb/image_raw" />
        <param name="pub_binary_topic" value="/road_segmentation/image_raw" />
    </node>  
</launch>
